sources/commands/option_external.c

```diff
@@ -49,13 +73,14 @@ int	option_external(t_control control, t_treenode *node, t_param *param)
 	*control.found = 1;
 	if (param->debug_output_level & DBG_EXTERNAL_SEARCH_FOLDERS)
 	{
-		printf("searching for command in folders:\ncurrent\n");
+		printf("searching for command in folders:\n");
 		ft_dlist_print_s(param->envvar_path_head, "\n");
 	}
-	fullpath = find_executable(node->content, param->envvar_path_head);
+	fullpath = find_executable(node->content, param->envvar_path_head,
+			&param->errno);
 	if (!fullpath)
```

Comment:
4. if the find_executable returned empty string, this command does not exist, fail, exit